packages/react-reconciler/src/ReactFiberTracingMarkerComponent.new.js

```diff
@@ -6,8 +6,64 @@
  *
  * @flow
  */
+
+import type {TransitionTracingCallbacks} from './ReactInternalTypes';
 import type {Fiber} from 'react-reconciler/src/ReactInternalTypes';
 
+export type SuspenseInfo = {name: string | null};
+
 export type Transition = {
+  name: string,
+  startTime: number,
+};
+
+export type BatchConfigTransition = {
+  name?: string,
+  startTime?: number,
   _updatedFibers?: Set<Fiber>,
 };
+
+export type Transitions = Set<Transition> | null;
```

Comment:
Maybe this should be an array instead of a Set? The total size is usually 1, maybe 2 or 3 sometimes, right?